### ISSUE TITLE
Retreating during the pause before a battle starts doesn’t cause an immediate retreat

### ISSUE THREAD

**Issue by dagwud** [dagwud/woodlands#76]
You encountered a Angry Raccoon (L1.0):
Angry Raccoon: 💚71 / 71, ⚔2.5 +2

IceKing buffed your defences by +1

You cast ✨Aura of Protection
—delay starts—
Village Square
—delay ends—
You are at The Village

⚔️ Battle Round #1: ⚔️

🔹IceKing ✨Aura of Protection
🔹HARRY! ✨Aura of Protection
🔸Angry Raccoon ⚔1+2 → HARRY!
🔹FUUUUUUCK 🏹18+4 → Angry Raccoon
🔹FUUUUUUCK 🏹21+15 → Angry Raccoon (🎯)
🔹IceKing 🏹11+3 → Angry Raccoon☠️
🔹Riverdog 😴 did nothing
🔹HARRY! 😴 did nothing
🔹JOHN! 😴 did nothing
🔹Sir Leaks a Lot 😴 did nothing

Stats after round 1

🔹JOHN!: 💚71 / 66 + 8, ✨3/4
🔹Riverdog: 💛74 / 87 + 8, ✨2/4
🔹Sir Leaks a Lot: 💚57 / 57, ✨0/4
🔹FUUUUUUCK: 💚51 / 52 + 8, ✨4/4
🔹HARRY!: 🧡13 / 26, ✨4/4
🔹IceKing: 💚24 / 29, ✨0/4
🔸Angry Raccoon: ☠️dead

**Comment by EvanKnowles** [comment]
It retreats, but doesn't end the battle, isn't it?

**Comment by dagwud** [comment]
Yeah, true. Will update

**Comment by dagwud** [comment]
Well... it does end the battle - but only after squeezing in one more round.

**Comment by dagwud** [comment]
Eliminated the delay as part of work for manual areas, so no longer relevant
